### ISSUE TITLE
代码复现问题

### ISSUE THREAD

**Issue by cqguowangshu** [zjunlp/deepke#43]
您好！我在学习您的deepke源代码，并试图复现相关实验，但是在使用Bert的时候（model=lm），一直报无法找到预训练模型的错误，错误代码为：Model name '/deepke-master/deepke-master/pretrained' was not found in tokenizers model name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased, bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking, bert-large-uncased-whole-word-masking-finetuned-squad, bert-large-cased-whole-word-masking-finetuned-squad, bert-base-cased-finetuned-mrpc, bert-base-german-dbmdz-cased, bert-base-german-dbmdz-uncased, bert-base-finnish-cased-v1, bert-base-finnish-uncased-v1, bert-base-dutch-cased). We assumed '/deepke-master/deepke-master/pretrained' was a path, a model identifier, or url to a directory containing vocabulary files named ['vocab.txt'] but couldn't find such vocabulary files at this path or url.
但是我是严格按照readme中的要求，下载了完整的bert模型（pytorch版）并放置在pretrained文件夹内，不知是何错误？
还有一个小问题，源代码中在predict里运行后有“实体类型”输入这一选项，但给出的train data格式中并没有“实体类型”，是否意味着我可以在训练数据中自行添加“实体类型”的训练特征，而后的预测也可以因此提高精确度？
请批评指正！望不吝赐教！谢谢！
![bug](https://user-images.githubusercontent.com/67032404/87946670-e873c480-cad4-11ea-9133-de50ef796b5e.jpg)

**Comment by yuwl798180** [comment]
你好，你的第一个问题是因为：你没有下载 'vocab.txt'，报错的最后一行。
第二个问题：当然可以的，输入信息给的越多，我们可使用的越多，自然性能越好。
举个例子，一般对于关系抽取，输入形式是： <e1>杭州</e1>是<e2>浙江</e2>的省会。
当有了entity type，你可以将句子预处理为：<e1-location>杭州</e1-location>是<e2-location>浙江</e2-location>的省会。
此时输入信号对于entity pair多了类型的信号，可以很大程度的增强模型的提取性能。

**Comment by cqguowangshu** [comment]
请问包含除实体位置外其他特征（如：实体类型）的训练数据格式是什么样的呀？能给一个范例csv或者表格形式吗？还是说需要我基于源码进行二次开发呢？

**Comment by yuwl798180** [comment]
你好，在`conf/preprocess.yaml`配置文件中，有相关配置。

```
# 是否需要使用实体类型替换实体词语
replace_entity_with_type: True

# 是否需要使用三元组头尾标记替换实体词语
replace_entity_with_scope: True
```
